src/test/java/edu/harvard/iq/dataverse/export/OpenAireExportUtilTest.java

```diff
@@ -609,6 +609,9 @@ public void testWriteRelatedIdentifierElement() throws XMLStreamException, IOExc
                 + "RelatedPublicationIDNumber1</relatedIdentifier>"
                 + "<relatedIdentifier relationType=\"IsCitedBy\" relatedIdentifierType=\"arXiv\">"
                 + "RelatedPublicationIDNumber2</relatedIdentifier>"
+                + "</relatedIdentifiers>"
```

Comment:
This line shouldn't be here. The related test is failing.

Reply:
Yes, because of the spelling error of xml format, I have finished revising it.